app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def create_file():
+    file_name = input("Input the name of file: ")
+    print("File content: ")
+
+    while True:
+        content = input()
+
+        if content.lower() == "stop":
+            break
+        else:
+            with open(file_name + ".txt", "a") as f:
```

Comment:
It's better to open the file once before the loop.
Opening the file for each line can take some extra time

Reply:
Got it